CHANGELOG.md

```diff
@@ -2,25 +2,29 @@
 
 <!-- latest_release unreleased -->
 ## Unreleased
+<!-- latest_release -->
+
+ ## [0.82.0](https://github.com/habitat-sh/habitat/tree/0.82.0) (2019-05-06)
+ [Full Changelog](https://github.com/habitat-sh/habitat/compare/0.81.0...0.82.0)
+
 
 #### Behavioral Changes
 - Add `--binlink-dir` argument to `hab pkg install` [#6597](https://github.com/habitat-sh/habitat/pull/6597) ([baumanj](https://github.com/baumanj))
 
 #### New Features & Enhancements
 - Run health checks asynchronously [#6482](https://github.com/habitat-sh/habitat/pull/6482) ([christophermaier](https://github.com/christophermaier))
+- Be more liberal in our license acceptance checking. [#6574](https://github.com/habitat-sh/habitat/pull/6574) ([raskchanky](https://github.com/raskchanky))
 
 #### Bug Fixes
 - remove trailing space from env vars set in export docker build [#6540](https://github.com/habitat-sh/habitat/pull/6540) ([mwrock](https://github.com/mwrock))
 
 #### Merged Pull Requests
 - Move deadlock error to trace - stop spam [#6609](https://github.com/habitat-sh/habitat/pull/6609) ([chefsalim](https://github.com/chefsalim))
-- Update error handling for origin deletion [#6467](https://github.com/habitat-sh/habitat/pull/6467) ([chefsalim](https://github.com/chefsalim))
```

Comment:
Why was this deleted?

Reply:
🤦‍♂  Because I don't know how to drive vim.

Reply:
@christophermaier fixed

Reply:
`vim` is a harsh master